homeassistant/components/teslemetry/sensor.py

```diff
@@ -623,17 +629,22 @@ async def async_added_to_hass(self) -> None:
         if (sensor_data := await self.async_get_last_sensor_data()) is not None:
             self._attr_native_value = sensor_data.native_value
 
+        assert self.entity_description.streaming_listener
```

Comment:
Let's avoid using the assert. This usually indicates an issue with typing.

../Frenck

Reply:
How can I make mypy happy then? This class only gets created when the property exists but mypy doesn't seem to understand that contextually.

Reply:
I've added an if statement instead of assert.